editoast/src/views/infra/errors.rs

```diff
@@ -151,4 +155,47 @@ mod tests {
         let response = call_service(&app, req).await;
         assert_eq!(response.status(), StatusCode::OK);
     }
+
+    #[rstest]
+    async fn get_errors_must_return_buffer_stop_errors(#[future] empty_infra: TestFixture<Infra>) {
+        let empty_infra = empty_infra.await;
+        let app = create_test_service().await;
+
+        let track_section: RailjsonObject = TrackSection {
+            id: "test_track".into(),
+            length: 1_000.0,
+            slopes: vec![Slope {
+                begin: 250.0,
+                end: 1250.0,
+                gradient: 0.,
+            }],
+            ..Default::default()
+        }
+        .into();
+
+        let create_operation = Operation::Create(Box::new(track_section));
+        let requeste = TestRequest::post()
+            .uri(format!("/infra/{}/", empty_infra.id()).as_str())
+            .set_json(json!([create_operation]))
+            .to_request();
+        call_service(&app, requeste).await;
+
+        let error_type = "missing_buffer_stop";
+        let level = "warnings";
```

Comment:
I don't if it's actually feasible, but it would be more robust to use somehow InfraErrorType::MissingBufferStop and Level::Warning to fill those values in